react/__mocks__/vtex.address-form/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { addValidation, removeValidation } from './helpers'
+import { StyleguideInput } from './inputs'
+
+const AddressRules = props => <div className="address-rules-mock" />
+
+const AddressContainer = props => <div className="address-container-mock" />
```

Comment:
Use css modules 🔨 .

Reply:
These `classNames`s are only used to make the snapshots better readable and not to customize the divs.

I checked in other tests and no one uses css modules, so I think that this confirms. Can you check again @ArthurSampaio ?